README.md

```diff
@@ -1,26 +1,266 @@
+  - toolboxes: turtle_son, etc...
+- **testDataSets**: benchtop generated test data sets for validation of code; often generated signals are simultaneously recorded with DAQ to allow for verification of timing across data streams. 
+- **outputFigs**: will contain the output of function testing to the specified test dataset
+
+## Functions: 
+This list contains the functions that have been tested in brach and pushed to master (brief description of function input output next to each function name)
+
+### Wrappers
+- **DEMO_ProcessRCS**: Demo wrapper script for importing raw .JSON files from RC+S, parsing into Matlab table format, and handling missing packets / harmonizing timestamps across data streams
+
+### CreateTables
+- **createDeviceSettingsTable**: Extract information from DeviceSettings related to configuration for time domain, power, and FFT channels
+- **createTimeDomainTable**: Create Matlab table of raw data from RawDataTD.json
+- **createAccelTable**: Create Matlab table of raw data from RawDataAccel.json
+- **createPowerTable**: Create Matlab table of raw data from RawDataPower.json
+- **createFFTtable**: Create Matlab table of raw data from RawDataFFT.json
+
+### Utility
+- **deserializeJSON**: Reads .json files and loads into Matlab
+- **fixMalfomedJSON**: Checks for and replaces missing brackets and braces in json file, which can prevent proper loading
+- **convertTDcodes**: Conversion of Medtronic numeric codes into values (e.g. Hz)
+- **getSampleRate**: Convert Medtronic codes to sample rates in Hz for time domain data
+- **getSampleRateAcc**: Convert Medtronic codes to sample rates in Hz for accelerometer data
+- **getPowerBands**: Calculate lower and upper bounds, in Hz, for each power domain timeseries
+- **getFFTparameters**: Determine FFT parameters from FFTconfig and TD sample rate
+
+### (Pre)Processing
+- **assignTime**: Function for creating timestamps for each sample of valid RC+S data. 
+
+## How to get a time value for each sample of data
+Ideally, there would be a value reported with each packet from which we could easily re-create unix time for each sample. Nominally, this would be PacketGenTime. However, upon inspection we see that: 
+- **(1)** The difference between adjacent PacketGenTime values does not always equal the expect amount of elapsed time, where the expected amount of elapsed time is calculated by the number of samples in the packet and the sampling rate. This is a serious problem. In cases where there is missing time, we would lose the stereotyped 1/Fs duration between samples. In cases of overlap, how do we account for having more than one value sampled at the same time?
+- **(2)** PacketGenTime has suspect values across packets and relative to other timing variables. Together, these indicate that PacketGenTimeis  unreliable when high precision time reconstruction is required (as for some task-locked analyses, looking at ERPs, etc).
```

Comment:
fix: PacketGenTime is

Reply:
Fixed with f66235b663ee7f630ad3e8aab9fda7a1ca4f639a